ncm-ntpd/src/main/pan/components/ntpd/config.pan

```diff
@@ -5,5 +5,14 @@
 
 unique template components/${project.artifactId}/config;
 
-include { 'components/${project.artifactId}/config-common' };
-include { 'components/${project.artifactId}/config-rpm' };
+include 'components/${project.artifactId}/schema';
+
+bind "/software/components/${project.artifactId}" = component_ntpd_type;
+
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+prefix '/software/components/${project.artifactId}';
+'active' ?= true;
+'dispatch' ?= true;
+'version' ?= '${no-snapshot-version}';
```

Comment:
If we have a use for them then leave them in. Just trying to make sure we have a consistent view of what is required in this files!